src/pint/simulation.py

```diff
@@ -514,25 +514,14 @@ def make_fake_toas_fromtim(
     --------
     :func:`make_fake_toas`
     """
-    ephem = (
-        model.EPHEM.value
-        if hasattr(model, "EPHEM") and model.EPHEM.value is not None
-        else None
-    )
-    planets = (
-        model.PLANET_SHAPIRO.value
-        if hasattr(model, "PLANET_SHAPIRO") and model.PLANET_SHAPIRO.value is not None
-        else False
-    )
+    ts = pint.toa.get_TOAs(timfile, model=model)
```

Comment:
So setting the model attribute will propagate BIPM, planets, etc, right?

Reply:
It should.